src/app/patient-dashboard/common/todays-vitals/test-todays-vitals.service.ts

```diff
@@ -0,0 +1,94 @@
+import { Injectable } from '@angular/core';
+
+import { TodaysVitalsService } from './todays-vitals.service';
+import { VitalsDatasource } from './vitals.datasource';
+
+@Injectable()
+/**
+ * TestTodaysVitalsService pretends to make real http requests.
+ * implements only as much of TodaysVitalsService as is actually consumed by the app
+*/
+export class TestTodaysVitalsService extends TodaysVitalsService {
+
+  constructor(_vitalsDataSource: VitalsDatasource) {
+    super(_vitalsDataSource);
+  }
+
+  lastResult!: Promise<any>;
+
+  todaysVitals1 = [
```

Comment:
why todayVitals1?

Reply:
Ideally I'd wanted to have two mock vitals - moving all the mock data into its own file solves this